outport/process/alteredaccounts/alteredAccountsProvider.go

```diff
@@ -125,6 +131,28 @@ func (aap *alteredAccountsProvider) processMarkedAccountData(
 	return nil
 }
 
+func (aap *alteredAccountsProvider) loadUserAccount(addressBytes []byte, options shared.AlteredAccountsOptions) (state.UserAccountHandler, error) {
+	var account vmcommon.AccountHandler
+	var err error
+
+	if !options.WithCustomAccountsRepository {
+		account, err = aap.accountsDB.LoadAccount(addressBytes)
+	} else {
+		account, _, err = options.AccountsRepository.GetAccountWithBlockInfo(addressBytes, options.AccountQueryOptions)
```

Comment:
[TODO]

Or have another func in this `AccountsRepository` interface that returns only `GetAccount`

Reply:
Probably this comment got lost

Reply:
we don't need an additional function